Formula/edirect.rb

```diff
@@ -25,10 +24,12 @@ def install
 
     ENV.prepend_create_path "PERL5LIB", prefix/"perl5/lib/perl5"
     ENV["OPENSSL_PREFIX"] = Formula["openssl"].opt_prefix # for Net::SSLeay
-    pms = %w[Encode::Locale File::Listing HTML::Parser HTML::Tagset HTML::Entities HTML::Tree
+    pms = %w[XML::Simple Encode::Locale File::Listing HTML::Parser HTML::Tagset HTML::Entities HTML::Tree
              HTTP::Cookies HTTP::Date HTTP::Message HTTP::Negotiate LWP::MediaTypes IO::Socket::SSL
              LWP::Protocol::https URI WWW::RobotRules Mozilla::CA Net::SSLeay]
-    system "cpanm", "--self-contained", "-l", prefix/"perl5", *pms
+    ENV.deparallelize
+    system "cpanm", "--force", "--self-contained", prefix/"perl5", "XML::SAX::Expat"
```

Comment:
```suggestion
    system "cpanm", "--force", "--self-contained", "-l", prefix/"perl5", "XML::SAX::Expat"
```

Reply:
When I was unsetting `MAKEFLAGS`, the `XML::SAX::Expat` installation would fail when I was testing it locally and included `-l `. I'll try including it with `ENV.deparallelize` now.